app/main.py

```diff
@@ -1,6 +1,27 @@
+import os
+
+import requests
+
+
+URL = "https://api.weatherapi.com/v1/current.json"
+API_KEY = os.environ.get("API_KEY")
+CITY = "Paris"
+PARAMS = {"key": API_KEY, "q": CITY}
```

Comment:
`PARAMS` is not looks like a const.
I think you should use it like a variable in function `get_weather()`

Reply:
I think it's not a mistake, so i will approve it